inspirehep/bat/pages/holding_panel_literature_list.py

```diff
@@ -36,7 +36,7 @@
 
 
 def go_to():
-    Arsenic().get(os.environ['SERVER_NAME'] + '/holdingpen/list/?page=1&size=10&source=submission&workflow_name=HEP&status=HALTED')
+    Arsenic().get(os.environ['SERVER_NAME'] + '/holdingpen/list/?page=1&size=10&status=HALTED')
```

Comment:
This looks suspicious: why are you removing `workflow_name=HEP` here? Of course, the better solution is to have proper test isolation by recreating the `holdingpen` tables, but meanwhile...